### DISCUSSION TITLE
[Feature] Better "(re)download to device" Behavior

### DISCUSSION THREAD

**Discussion by Vcele** [immich-app/immich#6645] [Feature Request]
### The feature

As already discussed in #4505 assets don't get re-downloaded to their original folder if deleted from the cloud. Instead of downloading them into the "Pictures" folder instead one could download them to a locally defined "cache" folder where all manually downloaded images are stored. Following this approach it would also be possible to directly link the photo and show the correct sync status, as well as providing the extended delete functions.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#11896
